### PR TITLE
fix(#3): updating how we find vans

### LINKED ISSUE #3: Issue with parsing when more than 3 vans are available
We make the assumption that there will only be one foodPark entry within each child div, however there can be multiple. This causes some options to be missed.

https://github.com/arylatt/foodpark/blob/main/main.go#L73 and https://github.com/arylatt/foodpark/blob/main/main.go#L76

### PR DESCRIPTION
**Pull request by arylatt**
Updating how we find vans to be url-first to avoid issues with different header styles and multiple vans per parent div.

Fixes #3.

### CODE REVIEW

**Comment by dantdj** [comment]
smh didn't even give me a chance to review

**Comment by arylatt** [comment]
> 
> 
> smh didn't even give me a chance to review

I couldn't actually add you as a reviewer because you aren't a collaborator on the repo.

**Comment by dantdj** [comment]
Booo

File: main.go [hunk 1]
```diff
@@ -69,23 +69,29 @@ func main() {
...
+				log.Fatal("Anchor tag does not have a href attribute")
+				os.Exit(1)
+			}
+			if !strings.Contains(url, viper.GetString("target_date")) {
+				log.Fatalf("Failed to find target date string %s in URL %s", viper.GetString("target_date"), url)
+				os.Exit(1)
+			}
 
-		foodParkOptions = append(foodParkOptions, foodOption)
+			name := s1.Parent().Parent().Parent().Prev().Text()
```
Comment (dantdj) [inline main.go:86]: Would be super cool if we could automate this a bit (I imagine you have similar thoughts), though with HTML trawling being what it is, maybe not 😅 I'll have a think

Comment (arylatt) [reply main.go:86]: Are you saying that having an extremely fragile assumption that the parents, children, and siblings aren't going to change is a bad idea? Impossible!

Comment (dantdj) [reply main.go:86]: These kinds of hard-hitting insights are why you're glad I'm looking at this right

**Pull request merged**